sympy/interactive/printing.py

```diff
@@ -330,28 +330,28 @@ def init_printing(pretty_print=True, order=None, use_unicode=None,
     >>> from sympy.abc import x, y
     >>> sqrt(5)
     sqrt(5)
-    >>> init_printing(pretty_print=True) # doctest: +SKIP
-    >>> sqrt(5) # doctest: +SKIP
+    >>> init_printing(pretty_print=True)
+    >>> sqrt(5)
       ___
     \/ 5
-    >>> theta = Symbol('theta') # doctest: +SKIP
-    >>> init_printing(use_unicode=True) # doctest: +SKIP
-    >>> theta # doctest: +SKIP
-    \u03b8
-    >>> init_printing(use_unicode=False) # doctest: +SKIP
-    >>> theta # doctest: +SKIP
+    >>> theta = Symbol('theta')
+    >>> init_printing(use_unicode=True)
+    >>> theta
+    θ
+    >>> init_printing(use_unicode=False)
+    >>> theta
     theta
-    >>> init_printing(order='lex') # doctest: +SKIP
-    >>> str(y + x + y**2 + x**2) # doctest: +SKIP
+    >>> init_printing(order='lex')
+    >>> str(y + x + y**2 + x**2)
     x**2 + x + y**2 + y
-    >>> init_printing(order='grlex') # doctest: +SKIP
-    >>> str(y + x + y**2 + x**2) # doctest: +SKIP
+    >>> init_printing(order='grlex')
+    >>> str(y + x + y**2 + x**2)
     x**2 + x + y**2 + y
-    >>> init_printing(order='grevlex') # doctest: +SKIP
-    >>> str(y * x**2 + x * y**2) # doctest: +SKIP
+    >>> init_printing(order='grevlex')
+    >>> str(y * x**2 + x * y**2)
     x**2*y + x*y**2
-    >>> init_printing(order='old') # doctest: +SKIP
-    >>> str(x**2 + y**2 + x + y) # doctest: +SKIP
+    >>> init_printing(order='old')
+    >>> str(x**2 + y**2 + x + y)
```

Comment:
Changing printing order affects subsequents doctests.

Reply:
But they are already skipped, would it still lead to failure?

Reply:
Doesn't this PR unskip it?

Reply:
This PR unskips this doctest. But the ones after this are skipped. You said for 'subsequent' doctests, which wouldn't be a problem I guess.

Reply:
I meant doctests in the subsequent files. `DocTestRunner` runs them in the same process.

Reply:
Okay, I can now relate to it. I can see that now in `sympy/utilities/runtests.py`.